projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +169,112 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function extractTicketDetailsFromFirstRowInTicketListingPage(): TestTicketDetails {
+  let testTicketDetails: TestTicketDetails = {
+    subject: "Temp",
+    message: "Temp",
+    category: TestCategory.complaint,
+    id: "000000",
+    status: "Open"
+  };
```

Comment:
No need to make it LET because its reference is not mutating, what is mutating are its internal values. So CONST should work.
Also, you can create an empty TestTicketDetails object and fill it with just the content you find.

```suggestion
  const testTicketDetails: TestTicketDetails;
```